src/PerfView/EventViewer/EventWindow.xaml.cs

```diff
@@ -1804,6 +1840,17 @@ private void SelectedCellsChanged(object sender, SelectedCellsChangedEventArgs e
             }
 
             m_maxColumnInSelection = null;
+
+            if (!cells.IsNullOrEmpty())
```

Comment:
```suggestion
            if (cells != null && cells.Count > 0)
```
`IsNullOrEmpty` is fairly heavy for this check because it goes through a LINQ abstraction and results in construction of an enumerator and an attempted enumeration operation, when all we really need to know if is there are any items in the collection  It's preferrable to avoid this sort of call when we can and check a bit more explicitly.

Reply:
Yes you're right. Just checking for null and any items makes more sense.